helm/kubernetes-cluster-autoscaler-chart/templates/rbac.yaml

```diff
@@ -0,0 +1,150 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ .Values.name }}
+  labels:
+    giantswarm.io/service-type: "{{ .Values.serviceType }}"
+    k8s-addon: cluster-autoscaler.addons.k8s.io
+    k8s-app: {{ .Values.name }}
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - events
+  - endpoints
+  verbs:
+  - create
+  - patch
+- apiGroups: [""]
```

Comment:
Are these the recommended permissions? I did not find anything in the upstream repo on a quick scan.

Reply:
I took most of this from the examples provided by the project: e.g. here https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/cloudprovider/aws/examples/cluster-autoscaler-run-on-master.yaml